### PR TITLE
fix bug.when create get/set dynamic variable block from set/get , lost the type.

### PR DESCRIPTION
**Pull request by duzc2**
when create get/set dynamic variable block from set/get ,
lost the type.

## The basics

- [x ] I branched from develop
- [x ] My pull request is against develop
- [x ] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Test Coverage

Tested on:
 Desktop Chrome

### CODE REVIEW

File: blocks/variables_dynamic.js [hunk 1]
```diff
@@ -102,11 +102,14 @@ Blockly.Constants.VariablesDynamic.CUSTOM_CONTEXT_MENU_VARIABLE_GETTER_SETTER_MI
     }
     var opposite_type;
     var contextMenuMsg;
+    var varType;
     if (this.type == 'variables_get_dynamic') {
       opposite_type = 'variables_set_dynamic';
+      varType = this.outputConnection.check_ ? this.outputConnection.check_[0] : "";
```
Comment (rachel-fenichel) [inline blocks/variables_dynamic.js]: Rather than getting the first item in the checks list, get the variable from the field and use its type.

Comment (duzc2) [reply blocks/variables_dynamic.js]: OK ~ done ~

**Comment by duzc2** [comment]
The Travis CI build failed
why?

**Comment by rachel-fenichel** [comment]
You can click on "details" next to "The Travis CI build failed" to get the Travis details.  Open up one of the travis runs and look at the logs--it will tell you what failed.

This failure was a lint failure: https://travis-ci.org/google/blockly/jobs/427504763#L1467

**Comment by duzc2** [comment]
thanks
all passed now

**Pull request merged**